### ISSUE TITLE
Merge CUDNN

### ISSUE THREAD

**Issue by hshindo** [hshindo/Merlin.jl#15]
path: src/cuda/cudnn
- [x] activation
- [x] softmax
- [x] convolution
- [x] dropout
- [ ] maxpooling
- [ ] rnn ＃gru
- [ ] lrn
- [ ] batchnorm
- [ ] DivisiveNormalization
- [ ] SpatialTfGridGenerator
- [ ] SpatialTfSampler
